### DISCUSSION TITLE
List of Unity Types and their Type IDs for 2023.2.20f1

### DISCUSSION THREAD

**Discussion by Kavukamari** [nesrak1/UABEA#355] [Q&A] [answered]
do you know how I can get a list of the type IDs so that I can update the type database of UABE? as far as I know it might be the only program that still has plugins to reimport a model back into an asset bundle, unless you know how I might convert a model into a dump or raw file? I'm not seeing the plugins on this edition, so I don't know if it's possible in UABEA

**Comment by nesrak1** [comment, marked as answer]
UABE does not have a plugin to import meshes. It can only export obj and dae.
